application/controllers/admin/surveyadmin.php

```diff
@@ -575,7 +575,8 @@ public function getAjaxQuestionGroupArray($surveyid)
                         $questionText = isset($question->questionl10ns[$baselang])
                             ? $question->questionl10ns[$baselang]->question
                             : '';
-                        $curQuestion['question_flat'] = viewHelper::flatEllipsizeText($questionText, true);
+                        $decodedQuestionText = htmlspecialchars_decode(viewHelper::flatEllipsizeText($questionText, true));
+                        $curQuestion['question_flat'] = strip_tags($decodedQuestionText);
```

Comment:
Since vue.js already encoded, you can just decode 
and put a comment
` // vue.js encpode and no filter to get raw text, then decode it`

Reply:
No need to remove "false" tag put by user.

Reply:
ok, so only missing would be the comment, right?